packages/commonwealth/server/webhookNotifier.ts

```diff
@@ -130,7 +130,7 @@ const send = async (models, content: WebhookContent) => {
       },
       include: [models.Profile],
     });
-    profile = await address.getProfile();
+    profile = await address.Profile;
```

Comment:
how can we trigger this endpoint locally @timolegros ?

Reply:
This is a webhook, not a WebSocket so I'm not sure how to trigger this. I assume some notification types will trigger this.

Reply:
```
  // send data to relevant webhooks
  if (
    webhook_data &&
    // TODO: this OR clause seems redundant?
    (webhook_data.chainEventType?.chain ||
      !erc20Tokens.includes(webhook_data.chainEventType?.chain))
  ) {
    await send(models, {
      notificationCategory: category_id,
      ...(webhook_data as Required<WebhookContent>),
    });
  }
```
Check `emitNotifications.ts`

Reply:
if not called that often i can skip it for now, as I would like other change in getAddress to go through

Reply:
removed